bookshelf/src/app.php

```diff
@@ -39,8 +39,11 @@
 // register the url generator
 $app->register(new UrlGeneratorServiceProvider);
 
+$config = getenv('BOOKSHELF_CONFIG') ?: 'settings.yml';
```

Comment:
It would be more intuitive to make the the full path to the file, rather than just the filename